components/script/dom/offscreencanvasrenderingcontext2d.rs

```diff
@@ -26,6 +29,7 @@ impl OffscreenCanvasRenderingContext2D {
         OffscreenCanvasRenderingContext2D {
             reflector_: Reflector::new(),
             canvas: canvas.map(Dom::from_ref),
+            canvas_state: DomRefCell::new(CanvasState::new(_global, _size)),
```

Comment:
The `_` prefix tells the compiler that these arguments are unused. Let's remove the prefix now.